js/redux/productReducer.js

```diff
@@ -8,4 +8,4 @@ export function productReducer(state= products, action) {
     
     default: return state;
   }
-}
\ No newline at end of file
+}
```

Comment:
Missed semicolon

Reply:
function declaration doesn't need a semicolon at the end (after curly brace)